packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,239 @@
+// @flow strict
+import { type Node, type Element, type Ref, useRef, useState, useEffect, useId } from 'react';
+import Box from './Box.js';
+import Layer from './Layer.js';
+import TapArea, { type OnTapType } from './TapArea.js';
+import Popover from './Popover.js';
+import Text from './Text.js';
+import Link, { type ExternalLinkIcon } from './Link.js';
+import Icon from './Icon.js';
+import Flex from './Flex.js';
+import Tooltip from './Tooltip.js';
+import styles from './HelpButton.css';
+import { type Indexable, CompositeZIndex, FixedZIndex } from './zIndex.js';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider.js';
+import { useColorScheme } from './contexts/ColorSchemeProvider.js';
+import { ESCAPE, SPACE, ENTER } from './keyCodes.js';
+
+type Props = {|
+  /**
+   * Supply a short, descriptive label to complete icon description `Click to learn more about [accessibilityLabel]`. Used for [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) purposes.
```

Comment:
Remove ``Click to learn more about [accessibilityLabel]`. Used for [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) purposes.`

Add, "See [Accessibility] section for guidance"

Reply:
Done